src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/PersonModal/PersonModal.test.tsx

```diff
@@ -0,0 +1,955 @@
+      userEvent.click(getByText('Show Less'));
+      userEvent.click(getByText('Save'));
+      await waitFor(() =>
+        expect(mockEnqueue).toHaveBeenCalledWith(
+          'Person updated successfully',
+          {
+            variant: 'success',
+          },
+        ),
+      );
+      const { operation } = mutationSpy.mock.calls[0][0];
+      expect(operation.variables.accountListId).toEqual(accountListId);
+      expect(
+        operation.variables.attributes.facebookAccounts[0].username,
+      ).toEqual(newPersonFacebookAccount);
+      expect(
+        operation.variables.attributes.twitterAccounts[0].screenName,
+      ).toEqual(newPersonTwitterAccount);
+      expect(
+        operation.variables.attributes.linkedinAccounts[0].publicUrl,
+      ).toEqual(newPersonLinkedInAccount);
+      expect(operation.variables.attributes.websites[0].url).toEqual(
+        newPersonWebsite,
+      );
+    });
+
+    it('should handle deleting socials', async () => {
+      const mutationSpy = jest.fn();
+      const { getByText, getAllByLabelText } = render(
+        <SnackbarProvider>
+          <MuiPickersUtilsProvider utils={LuxonUtils}>
+            <ThemeProvider theme={theme}>
+              <GqlMockedProvider<UpdatePersonMutation> onCall={mutationSpy}>
+                <PersonModal
+                  contactId={contactId}
+                  accountListId={accountListId}
+                  handleClose={handleClose}
+                  person={mockPerson}
+                />
+              </GqlMockedProvider>
+            </ThemeProvider>
+          </MuiPickersUtilsProvider>
+        </SnackbarProvider>,
+      );
+      expect(getByText('Edit Person')).toBeInTheDocument();
+      userEvent.click(getByText('Show More'));
+      userEvent.click(getAllByLabelText('Modal Section Delete Icon')[6]);
+      userEvent.click(getAllByLabelText('Modal Section Delete Icon')[8]);
+      userEvent.click(getAllByLabelText('Modal Section Delete Icon')[10]);
+      userEvent.click(getAllByLabelText('Modal Section Delete Icon')[12]);
+
+      userEvent.click(getByText('Save'));
+      await waitFor(() =>
+        expect(mockEnqueue).toHaveBeenCalledWith(
+          'Person updated successfully',
+          {
+            variant: 'success',
+          },
+        ),
+      );
+      const { operation } = mutationSpy.mock.calls[0][0];
+      expect(operation.variables.accountListId).toEqual(accountListId);
+      expect(
+        operation.variables.attributes.facebookAccounts[1].destroy,
+      ).toEqual(true);
+      expect(operation.variables.attributes.twitterAccounts[1].destroy).toEqual(
+        true,
+      );
+      expect(
+        operation.variables.attributes.linkedinAccounts[1].destroy,
+      ).toEqual(true);
+      expect(operation.variables.attributes.websites[1].destroy).toEqual(true);
+    });
+  });
+
+  describe('Creating', () => {
+    it('should handle creating a new person', async () => {
```

Comment:
@OzzieOrca This is the test that is having issues. As I said, the test works fine and I'm able to test all functionality except the cache update logic.